<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>929</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutMain">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="MyWidget" name="widget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>600</width>
         <height>600</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout1">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout1">
           <item>
            <widget class="QLabel" name="labelDeathUp">
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>12</pointsize>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>Death Upper</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelDeathLow">
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Death Lower</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelBirth">
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Birth</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer1">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Expanding</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="labelSpeed">
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Speed</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout2">
           <item>
            <widget class="QLineEdit" name="lineEditDeathUp">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>4</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditDeathLow">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>1</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditBirth">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>3</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Expanding</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditSpeed">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>200</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonClear">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyWidget</class>
   <extends>QWidget</extends>
   <header>widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
